src/gradleperfsmoketest.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+gradleperfsmoketest () {
```

Comment:
What do you think about calling this gradle[int/onpr/testrunner]smoketest or something else instead? Minor name change, but the perf part seems a little confusing since we're not actually running against the perf environment.

Reply:
Very good point - Does gradletestrunnersmoketest work?